File: Programmers/C#/lv1/lv1_이상한 문자 만들기.cs
https://programmers.co.kr/learn/courses/30/lessons/12930?language=csharp

public class Solution
{
    public string solution(string s)
    {
        string answer = "";

        int ctr = 0;
        foreach (char c in s)
        {
            answer += (ctr % 2 == 0) ? c.ToString().ToUpper() : c.ToString().ToLower();
            ctr = (c.Equals(' ')) ? 0 : ctr + 1;
        }

        return answer;
    }
}

File: Programmers/lv1/lv1_이상한 문자 만들기.java
https://programmers.co.kr/learn/courses/30/lessons/12930

import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        String[] str = s.split("");
        
        int count = 0;
        for(int i = 0; i < str.length; i++){
            if(str[i].equals(" ")){
                count = 0;
            }
            else if (count%2 == 0){
                str[i] = str[i].toUpperCase();
                count++;
            }
            else{
                str[i] = str[i].toLowerCase();
                count++;
            }
            answer += str[i];
        }
        
        return answer;
    }
}